python/lsst/pipe/tasks/calibrate.py

```diff
@@ -632,12 +659,16 @@ def getSchemaCatalogs(self):
         return {"src": sourceCat}
 
     def setMetadata(self, exposure, photoRes=None):
-        """!Set task and exposure metadata
+        """Set task and exposure metadata
 
         Logs a warning and continues if needed data is missing.
 
-        @param[in,out] exposure  exposure whose metadata is to be set
-        @param[in]  photoRes  results of running photoCal; if None then it was
+        Parameters
+        ----------
+        exposure :
+            exposure whose metadata is to be set
+        photoRes :
+            results of running photoCal; if None then it was
```

Comment:
Please double check for extraneous line breaks like this one (it's definitely fine for lines to be 79 characters long).